search-app/docs/api.md

```diff
@@ -2473,6 +2474,48 @@ __Example Response__
 }
 ```
 
+### <a name="deleted-collections"></a> Find collections that have been deleted
+
+To support metadata harvesting, a harvesting client can search CMR for collections that are deleted within a time period. The only search parameter supported is `revision_date` and its format is the same as the `revision_date` parameter in regular collection search. The only supported result format is xml references. The response is the references to the highest non-tombstone collection revisions of the collections that are deleted within the given revision date period. e.g.
```

Comment:
How much work would it be to support all of the same formats as the collections route?